hawc/apps/assessment/templates/assessment/assessmentvalues_detail.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'assessment-rooted.html' %}
+
+{% block content %}
```

Comment:
For now, can we revise this template to look more like the /study/:id template?

That is, there will be an Actions button in the top right to update/delete, and then the attributes of the value will be in a table, where each row is an attribute.

That way it'll be easier to add new values